x-pack/plugins/observability_solution/observability_ai_assistant/public/plugin.tsx

```diff
@@ -60,14 +61,15 @@ export class ObservabilityAIAssistantPlugin
     coreStart: CoreStart,
     pluginsStart: ObservabilityAIAssistantPluginStartDependencies
   ): ObservabilityAIAssistantPublicStart {
+    const scopeFromUiSettings = coreStart.uiSettings.get(AI_ASSISTANT_DEFAULT_SCOPE_KEY);
     const service = (this.service = createService({
       analytics: coreStart.analytics,
       coreStart,
       enabled:
         coreStart.application.capabilities.observabilityAIAssistant[
           aiAssistantCapabilities.show
         ] === true,
-      scope: this.scopeFromConfig || 'observability',
+      scope: this.scopeFromConfig || scopeFromUiSettings || 'search',
```

Comment:
why does this default to `search`? I think we discussed this in the meeting, but I asked if we can do this based on data (e.g., are there any data streams matchings logs-*,metrics-* or traces etc).

Reply:
IIRC we discussed using data by deferred that to a later stage.

Reply:
Ok, I would like to leave it at `observability` as a forcing function for us to actually use better signals to determine the use case